### ISSUE TITLE
VERIFY QUERY REQUEST: search_peptides

### ISSUE THREAD

**Issue by kaimikacolin** [oceanproteinportal/api#17]
@jaclynsaunders @ashepherd Please check this query and verify the snippets I've assembled are valid:

[Complete Peptide Search Query](https://kg.oceanproteinportal.org/?query=SELECT+DISTINCT+STR%28%3Fdepth%29+AS+%3Fdepth+%3Fstation+STR%28%3FspectralCount%29+AS+%3FspectralCount%0D%0A++++++++WHERE+%0D%0A++++++++%7B+%0D%0A++++++++++++%3Curn%3Aopp%3Adataset%3Anunn-bering-sea%3E+opp%3AstoresResultsForSample+%3Fsample+.+%0D%0A%3Fprotein+view%3AfilterSize+%3Chttp%3A%2F%2Fid.oceanproteinportal.org%2Ffilter-size%2F285a3e49-9384-3958-8617-bba735f58571%3E+.+%0D%0A%0D%0A++++++++++++%0D%0A++++++++++++%3Fsample+sosa%3AisResultOf+%3Fsample_event+.+%0D%0A++++++++++++%3Fsample_event+sosa%3AresultTime+%3Fdate_time+.+%0D%0A++++++++%0D%0AFILTER+%28%3Fdate_time+%3E%3D+%222015-08-31%22%5E%5Exsd%3AdateTime%29+.+%0D%0AFILTER+%28%3Fdate_time+%3C%3D+%222022-07-09%22%5E%5Exsd%3AdateTime%29+.+%0D%0A%0D%0A++++++++++++%3Fsample+sosa%3AisResultOf+%3Fsample_event+.%0D%0A++++++++++++%3Fsample_feature+opp%3Adepth+%3Fdepth+.+%0D%0A++++++++%0D%0AFILTER+%28%3Fdepth+%3E%3D+1380%29+.+%0D%0AFILTER+%28%3Fdepth+%3C%3D+4844%29+.+%0D%0A%0D%0A++++++++++++%3Fsample+sosa%3AisSampleOf+%3Ffeature+.%0D%0A++++++++++++%3Ffeature+opp%3Adepth+%3Fdepth+.%0D%0A++++++++++++%3Ffeature+opp%3AinVicinityOfStation+%3FstationInfo+.%0D%0A++++++++++++%3FstationInfo+opp%3AstationName+%3Fstation+.+%09+%0D%0A++++++++++++%3Fobservation+opp%3AobservationInSample+%3Fsample+.%0D%0A++++++++++++%3Fobservation+schema%3Avalue+%3FspectralCount+.+%0D%0A++++++++++++%3Fobservation+rdf%3Atype+opp%3APeptideSpectralCountSum+.%0D%0A++++++++++++%3Fpeptide+opp%3AsamplePeptideProperty+%3Fobservation+.%0D%0A++++++++++++%3Fpeptide+opp%3AdescribesPeptide+%3Fidentifier+.%0D%0A++++++++++++%3Fidentifier+opp%3ApeptideSequence+%22AAAAELAAFK%22%5E%5Exsd%3Atoken+.%0D%0A++++++++++++%7D%0D%0A++++++++ORDER+BY+%3Fstation%0D%0A++++%0D%0A)

**Comment by jaclynsaunders** [comment]
@kaimikacolin - this query doesn't seem to be returning anything. Was it working for you before?

**Comment by kaimikacolin** [comment]
No, I composed this as best I could from snippets, but as they were sourced from Protein queries, so I think they will need to change for peptides.
